<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnoopConfigDialog</class>
 <widget class="QDialog" name="SnoopConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JPEGsnoop Confguration</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>402</width>
     <height>406</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Directory for User Database</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>381</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="leDbPath">
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="btnBrowse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDefault">
            <property name="text">
             <string>Default</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Software Update</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="cbAutoUpdate">
          <property name="maximumSize">
           <size>
            <width>260</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Automatically check for updates every </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leUpdateDays">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>days</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Processing Options</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>335</width>
         <height>42</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="cbAutoReprocess">
          <property name="text">
           <string>Auto Reprocess files on change in Options (slower)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbOblineDb">
          <property name="text">
           <string>Submit signature to Online DB when User submit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Report Settings</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>325</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Max # errors to report in Scan Decode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leMaxErrors">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Help</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>311</width>
         <height>48</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Coach Messages can be re-enabled:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReset">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
